stable/mlflow/values.yaml

```diff
@@ -14,7 +14,8 @@ extraLabels: {}
 
 service:
   type: ClusterIP
-  port: 5000
+  port: 80
+  targetPort: 5000
```

Comment:
since the target port and deployment port is the same value i will recommend using only one will reduce possible errors that might be caused if they different values.

Reply:
makes sense, fixed.